src/style.css

```diff
@@ -0,0 +1,20 @@
+html {
+  font-size: 13px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: auto;
```

Comment:
As task description says:
`Remove all the outer spacing for all the heading and paragraphs.`
Let's add some rules for this too